pkg/infra/block_collector.go

```diff
@@ -39,8 +44,14 @@ func (bc *BlockCollector) Start(
 	now time.Time,
 	printResult bool, // controls whether to print block commit message. Tests set this to false to avoid polluting stdout.
 ) {
+	bc.status = 1
```

Comment:
what's this for? if we **have to** introduce an enum here (i hope not), please declare them